common/functions/src/scalars/others/running_difference_function.rs

```diff
@@ -51,7 +51,7 @@ impl Function for RunningDifferenceFunction {
 
     fn return_type(&self, args: &[DataTypeAndNullable]) -> Result<DataTypeAndNullable> {
         let nullable = args.iter().any(|arg| arg.is_nullable());
-        let data_type = match args[0].data_type() {
+        let data_type = match args[0].data_type().remove_nullable() {
```

Comment:
Maybe we should not use `remove_nullable`, if there is null, it will throw error result.

Reply:
Right, the next PR I am going to address will change return_type to ```return_type(&self, args: &[DataType])``` .  For now, nullable data type is directly passed into this function,  thus the remove_nullable happens inside of the function. I will change the logic to move remove_nullable outside of function.rs in the next PR.